include/libtorrent/create_torrent.hpp

```diff
@@ -347,7 +347,7 @@ namespace libtorrent
 		// and they have the same name.
 		bool m_multifile:1;
 
-		// this is true if the torrent is private. i.e., is should not
+		// this is true if the torrent is private. i.e., its interested peers should not
```

Comment:
Thank you, now I have a better understanding and agree: it should be "this client".
And no, I meant simply "other peers" with "interested peers".

Shall I close these pull requests and create new ones with "this client"?

90 lines above in `create_torrent.hpp`, I just found the comment for `set_priv()` is saying it perfectly: "... not advertise itself publicly...,"
Why not write "see comment for set_priv()" ?